config/api.yml

```diff
@@ -68,15 +65,6 @@
         :identifier: miq_action_edit
       - :name: delete
         :identifier: miq_action_delete
-      :patch:
-      - :name: edit
-        :identifier: miq_action_edit
-      :put:
-      - :name: edit
-        :identifier: miq_action_edit
```

Comment:
@kbrock are these product features still used when you POST with an action of edit?

Reply:
we have never used put/edit, or patch/edit, we always use post/edit.
For all these examples, there should remain a post/edit. If not, then I introduced a bug

Reply:
let me try again

Yes. We use `miq_action_edit` when we `PUT/edit` and `POST/edit`.
We are currently, and have always, used `POST/edit` privileges for both.